awesome_owl/static/src/todo_list/todo_list.js

```diff
@@ -0,0 +1,38 @@
+import { Component, useState } from "@odoo/owl";
+import { TodoItem } from "./todo_item";
+import { useAutofocus } from "../utils";
+
+export class TodoList extends Component {
+    static template = "awesome_owl.TodoList";
+    static components = { TodoItem };
+
+    setup() {
+        this.nextId = 1;
+        this.todos = useState([]);
+        useAutofocus("input")
+    }
+
+    addTodo(ev) {
+        if (ev.keyCode === 13 && ev.target.value != "") {
+            this.todos.push({
+                id: this.nextId++,
+                description: ev.target.value,
+                isCompleted: false
+            });
+            ev.target.value = "";
+        }
+    }
+
+    toggleTodo(todoId) {
+        const todo = this.todos.find((todo) => todo.id === todoId);
```

Comment:
Destructuring is your friend
```suggestion
        const todo = this.todos.find(({ id }) => id === todoId);
```